README.md

```diff
@@ -59,6 +59,10 @@ const userOptions = {
 };
 
 await ciscoTracing.init(userOptions);
+
+// The instrument requires should be called after ciscoTracing init
```

Comment:
I suggest: The import of you instrumented library should go her, **after** the ciscoTracing.init()

Reply:
Done